_src/_posts/2018-01-04-importance-of-serving-ads.markdown

```diff
@@ -124,51 +120,43 @@ These relationships were created from the data, yet we could not explicitly give
 
 We repeat the series above with another campaign, this is shown in the figure below.
 
-![ Side comparison of both CPA and revenue series monotonous trade off for a different campaign.]({{site.url}}/assets/images/frequency-capping/cpa_revenue_comparison_chart.png){: .center-image }
+![ Side comparison of both CPA and volume series monotonous trade off for a different campaign.]({{site.url}}/assets/images/frequency-capping/cpa_revenue_comparison_chart.png){: .center-image }
 
-Once again we see how the CPA is at maximum levels when the caps are at their minimum. Note how in this case we have a different functional relationship between CPA and cap level, compared to the previous figure. The same goes with the revenue decrease.
+Once again we see how the CPA is at maximum levels when the caps are at their minimum. Note how in this case we have a different functional relationship between the CPA and the cap level, in comparison to the previous figure. The same can be said for the volume decrease.
 
 The question now remains: where do we set the optimal frequency cap, given these two metrics? This is a broad question which is more dependant on how the company values them. For our case, we decided to value both equally. We think this is both advantageous for us and our clients.
 
 
 Given this multi-objective optimization setting, we scalarized the values in a single functional form. Valuing both equally means setting the same weights for both. Thus we find the optimal frequency cap level \\(f \in F\\) by choosing:
 
 $$
-\mathrm{argmax}_{f \in F} Cpa(f) + Rev(f)
+\mathrm{argmax}_{f \in F} Cpa(f) + Cl(f)
 $$
 
 as our optimal cap level.
 
 Again, this valuation is something that suits our way of understanding the business. Different valuations create different optimization forms. We tried other ways, such as Pareto optimal relations, or other \\(\epsilon\\)-constrained methods. You can find more cool stuff about this topic on [Wikipedia][wiki-scalarization][^3].
 
-The previous optimization yielded results which were more than satisfying. In general, we found optimal daily frequency caps per users to be around five to twenty impressions. This makes sense if we think that when we want to get an advertising message across to a large group of people. We won't get them immediately to convert, but, in general, we have that there is a amount of impressions which is just _enough_.
+The previous optimization yielded results which were more than satisfying. In general, we found that the optimal number of daily frequency caps per user was in between five to twenty impressions. This makes sense if we think that, when we want to get an advertising message across to a large group of people, we won't get them immediately to convert. However, in general we have that there is an amount of impressions which is just _enough_.
```

Comment:
However, in general there is an amount of impressions which is just _enough_.

Reply:
Lol again. Hopefully lesson learnt.

 Should've just `s/\bwe\b\ \bhave\b\ \bthat\b//g`